packages/p2p/src/pages/advertiser-page/advertiser-page.jsx

```diff
@@ -149,12 +149,16 @@ const AdvertiserPage = () => {
         },
     });
 
-    const { error, isError, isLoading } = useP2PAdvertiserAdverts();
```

Comment:
This page only displays `info` from either `advertiser_page_store` or `general_store` as seen in the screenshot below. The only thing I should be getting from the `useP2PAdvertiserAdverts` is the `error` object, as this is not handled anywhere in `AdvertiserPageAdverts` and should be displayed on the `AdvertiserPage`

<img width="878" alt="advertiser-page jsx " src="https://github.com/binary-com/deriv-app/assets/104334373/08a84465-14b5-4554-b22b-e59e9227fd54">